Source/WebCore/dom/Document.h

```diff
@@ -593,6 +594,9 @@ class Document
     const ExtensionStyleSheets& extensionStyleSheets() const { return *m_extensionStyleSheets; }
 
     const Style::CustomPropertyRegistry& customPropertyRegistry() const;
+    const CSSCounterStyleRegistry& counterStyleRegistry() const;
+    CSSCounterStyleRegistry& counterStyleRegistry();
+    // CSSCounterStyleRegistry* counterStyleRegistry();
```

Comment:
We generally don't leave code commented. Please either delete it or uncomment it.